migration/20160913-2-change-all-license-pools-sources-to-internal-processing.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE licensepools SET data_source_id = (
+    SELECT id FROM datasources WHERE name = 'Library Simplified Internal Process'
+) WHERE id IS NOT NULL;
```

Comment:
This will turn all licensepools on the metadata wrangler into Internal Process license pools, thus divorcing the Metadata Wranglers license pools from any semblance of licensing. This gets rid of information about where the existing licensepool came from, but the existing license pool has no relevant meaning, so perhaps it's okay?